Content/Items/Moonstone/Weapons.CrescentQuarterstaff.cs

```diff
@@ -523,6 +525,7 @@ private void TentativelyExecuteSlam()
 
 				CameraSystem.shake += 16;
 				Projectile.NewProjectile(Projectile.GetSource_FromThis(), StaffEnd, Vector2.Zero, ProjectileType<GravediggerSlam>(), 0, 0, Player.whoAmI);
```

Comment:
this will create multiple projectiles in multiplayer if its not wrapped in logic that makes it only 1 client or the server